daemon/algod/api/spec/v1/model.go

```diff
@@ -169,6 +169,7 @@ type Transaction struct {
 	// To add another one, create a struct with XXXTransactionType and embed it here.
 	// To prevent extraneous fields, all must have the "omitempty" tag.
 	Payment *PaymentTransactionType `json:"payment,omitempty"`
+	Keyreg  *KeyregTransactionType  `json:"keyreg,omitempty"`
```

Comment:
While we're at it, can you swagger annotate both the `Payment` and `Keyreg` fields with a description, and `required: false`?

Reply:
Hi @Vervious we are currently looking at it. We were trying to figure out how the swagger file is generated. Thanks for the hint.

Reply:
Thanks! Let me know if you have any questions. We use the go-swagger annotations, which are a bit esoteric. The first line in the comment translates into a "Description" field